components/dashboard/src/settings/PageWithSettingsSubMenu.tsx

```diff
@@ -17,9 +18,14 @@ export interface PageWithAdminSubMenuProps {
 
 export function PageWithSettingsSubMenu({ title, subtitle, children }: PageWithAdminSubMenuProps) {
     const { userBillingMode } = useContext(UserContext);
+    const { showPersonalAccessTokens } = useContext(FeatureFlagContext);
```

Comment:
[nit] The feature flag should be used to control both the UI, and also the Backend. For the backend the keyword "show" doesn't make as much sense. Normally, I'd use `enable` as the keyword

Reply:
Good point! Will change.